src/emulator/gui/src/trophy_unlocked.cpp

```diff
@@ -0,0 +1,126 @@
+    if (gui.trophy_window_frame_stage == TrophyAnimationStage::SLIDE_IN
+         || gui.trophy_window_frame_stage == TrophyAnimationStage::SLIDE_OUT) {
+        ImVec2 target_window_pos = ImVec2(0.0f, 0.0f);
+
+        if (gui.trophy_window_frame_stage == TrophyAnimationStage::SLIDE_IN)
+            target_window_pos = ImVec2(ImGui::GetIO().DisplaySize.x - TROPHY_WINDOW_SIZE.x - TROPHY_WINDOW_MARGIN_PADDING,
+                TROPHY_WINDOW_Y_POS);
+        else
+            target_window_pos = ImVec2(ImGui::GetIO().DisplaySize.x + TROPHY_WINDOW_MARGIN_PADDING, TROPHY_WINDOW_Y_POS);
+
+        if (gui.trophy_window_frame_stage == TrophyAnimationStage::SLIDE_IN && gui.trophy_window_frame_count == 0) {
+            gui.trophy_window_pos = ImVec2(ImGui::GetIO().DisplaySize.x + TROPHY_WINDOW_MARGIN_PADDING, TROPHY_WINDOW_Y_POS);
+
+            // Load icon
+            gui.trophy_window_icon = load_image(gui, (const char *)callback_data.icon_buf.data(),
+                static_cast<std::uint32_t>(callback_data.icon_buf.size()));
+        } else if (gui.trophy_window_frame_stage == TrophyAnimationStage::SLIDE_IN && gui.trophy_window_pos.x > target_window_pos.x) {
+            gui.trophy_window_pos.x -= TROPHY_MOVE_DELTA;
+        } else if (gui.trophy_window_frame_stage == TrophyAnimationStage::SLIDE_OUT && gui.trophy_window_pos.x < target_window_pos.x) {
+            gui.trophy_window_pos.x += TROPHY_MOVE_DELTA;
+        } else {
+            gui.trophy_window_frame_stage = static_cast<TrophyAnimationStage>(static_cast<int>(gui.trophy_window_frame_stage) + 1);
+            gui.trophy_window_frame_count = 0;
+        }
+
+        gui.trophy_window_frame_count++;
+    }
+
+    ImGui::SetNextWindowBgAlpha(0.9f);
+    ImGui::SetNextWindowPos(gui.trophy_window_pos);
+    ImGui::PushStyleVar(ImGuiStyleVar_WindowBorderSize, 0.0f);
+    ImGui::PushStyleVar(ImGuiStyleVar_WindowRounding, 10.0f);
+    ImGui::PushStyleColor(ImGuiCol_WindowBg, ImVec4(0.76f, 0.75f, 0.76f, 1.0f)); // Smooth gray
+    ImGui::SetNextWindowSize(TROPHY_WINDOW_SIZE);
+    ImGui::Begin("##NoName", nullptr, ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoScrollbar | ImGuiWindowFlags_AlwaysAutoResize);
+
+    ImGui::SetWindowFontScale(1.3f);
+    ImGui::SetCursorPos(ImVec2(TROPHY_WINDOW_MARGIN_PADDING, TROPHY_WINDOW_MARGIN_PADDING));
+    ImGui::Columns(2, nullptr, false);
+    ImGui::SetColumnWidth(0, TROPHY_WINDOW_ICON_SIZE + TROPHY_WINDOW_MARGIN_PADDING * 2);
+    ImGui::Image((ImTextureID)gui.trophy_window_icon, ImVec2(TROPHY_WINDOW_ICON_SIZE, TROPHY_WINDOW_ICON_SIZE));
+    ImGui::PushStyleColor(ImGuiCol_Text, ImVec4(0.24f, 0.24f, 0.24f, 1.0f));
+    ImGui::SetColumnWidth(1, ImGui::GetIO().DisplaySize.x - ImGui::GetCursorPosX() - TROPHY_WINDOW_MARGIN_PADDING);
+    ImGui::NextColumn();
+
+    std::string trophy_kind_s = "?";
+
+    switch (callback_data.trophy_kind) {
+    case emu::np::trophy::TrophyType::PLATINUM: {
+        trophy_kind_s = "P";
```

Comment:
One last thing. Could this be the name of the rarity. E.g. "Bronze" instead of "B." I was a bit confused at first.

Reply:
It was too long to display on the screen, so I don't think I will

Reply:
Rpcs3 using full name but yes for user is more clear ^^

Reply:
ok, if you guys want!